lizard_connector/connector.py

```diff
@@ -470,6 +475,30 @@ def __init__(self, base="https://demo.lizard.net", username=None,
                 each endpoint parse call.
         """
         self.api_version = version
+        # API Deprecation warning
+        if self.api_version == 2:
+            depracationWarning ="""
```

Comment:
In python we camel case only for classes.

```suggestion
            deprecation_warning ="""
```